client/response_test.go

```diff
@@ -537,4 +537,134 @@ func Test_Response_Save(t *testing.T) {
 		err = resp.Save(nil)
 		require.Error(t, err)
 	})
+
+	t.Run("streaming with io.Writer without closing", func(t *testing.T) {
+		t.Parallel()
+
+		server := startTestServer(t, func(app *fiber.App) {
+			app.Get("/stream", func(c fiber.Ctx) error {
+				return c.SendString("streaming data")
```

Comment:
This is not streaming the server response. Ths fiber server needs to run with the streaming option enabled

Reply:
@Abhirup-99